itou/eligibility/models/iae.py

```diff
@@ -24,7 +24,7 @@ class EligibilityDiagnosisQuerySet(CommonEligibilityDiagnosisQuerySet):
     def for_job_seeker(self, job_seeker):
         return self.filter(job_seeker=job_seeker)
 
-    def for_job_seeker_and_siae(self, *, job_seeker, siae=None):
+    def for_job_seeker_and_siae(self, job_seeker, siae=None):
```

Comment:
On est un peu dans un entre-deux là, soit on se dit que les paramètres n'ont pas a être nommés (car effectivement ça fait beaucoup de redite :sweat_smile:) ou alors vu que `siae` est optionnel on demande le nom et donc la signature devrais plutôt être celle-ci :
```suggestion
    def for_job_seeker_and_siae(self, job_seeker, *, siae=None):
```